packages/integration-sdk-runtime/src/synchronization/index.ts

```diff
@@ -261,11 +261,6 @@ export async function finalizeSynchronization({
     );
   }
 
-  logger.info(
-    { synchronizationJob: finalizedJob },
-    'Synchronization finalization result.',
-  );
-
```

Comment:
I debated keeping this. The other synchronization job functions `synchronizationStatus` and `synchronizationAbort` don't log their own results. If this goes through I'll likely add it back in inside the managed integration.

I do end up logging the job but in `run.ts` instead.